doc/rtd/howto/rerun_cloud_init.rst

```diff
@@ -64,33 +64,10 @@ a result.
 
 .. _partially_rerun_cloud_init:
 
-How to partially re-run cloud-init
```

Comment:
I think this information is still very useful for debugging. I've typed this out manually to various different people at least a dozen times. Add a million warnings around it if you want or change it to `cloud-init --all-stages`, but I think we should keep it.

Reply:
If we're calling it deprecated and planning on eventual removal, I'd rather not keep the old commands around.

> Add a million warnings around it if you want or change it to cloud-init --all-stages, but I think we should keep it.

I'll go that route.